apps/prairielearn/src/pages/instructorAssessmentOverrides/instructorAssessmentOverrides.js

```diff
@@ -0,0 +1,104 @@
+const express = require('express');
+const asyncHandler = require('express-async-handler');
+const router = express.Router();
+const {loadSqlEquiv} = require('@prairielearn/postgres');
+const sqldb = require('@prairielearn/postgres');
+const sql = loadSqlEquiv(__filename);
+
+// Set the current timestamp for created_at field
+function convertToTimeZone(utcDate, timeZone) {
+  const options = { timeZone, year: 'numeric', month: '2-digit', day: '2-digit', hour: '2-digit', minute: '2-digit', second: '2-digit' };
+  return new Date(new Intl.DateTimeFormat('en-US', options).format(utcDate)).toISOString().slice(0, 16);
+  }
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    const params = {
+      assessment_id: res.locals.assessment.id,
+      current_student_uid: res.locals.user.uid,
+      current_assessment_title : res.locals.assessment.title
+    };
+    const result = await sqldb.queryAsync(sql.select_assessment_access_policy, params);
+    res.locals.policies = result.rows
+    // console.log(convertToTimeZone(new Date() , res.locals.course_instance.display_timezone))
+    // console.log("Res: " + JSON.stringify(res.locals.course_instance));
+    res.render(__filename.replace(/\.js$/, '.ejs'), {
+      policies: res.locals.policies,
+      assessment_id: params.assessment_id,
+      current_student_uid: params.current_student_uid,
+      current_assessment_title: params.current_assessment_title,
+      created_at: convertToTimeZone(new Date() , res.locals.course_instance.display_timezone)
+    });
+  })
+);
+
+router.post(
+  '/',
+  asyncHandler(async (req, res) => {
+    if (req.body.__action === 'add_new_override') {
+    const params = {
+      assessment_id: res.locals.assessment.id,
+      created_at: convertToTimeZone(new Date() , res.locals.course_instance.display_timezone),
```

Comment:
No need to jump through these hoops; just use `NOW()` in the insertion query and Postgres will use the correct time. You could even declare the default value for the `created_at` column to be `NOW()`, which we do in many of our tables.